src/Native/System.Native/pal_process.cpp

```diff
@@ -417,3 +427,51 @@ extern "C" char* GetCwd(char* buffer, int32_t bufferSize)
 
     return getcwd(buffer, UnsignedCast(bufferSize));
 }
+
+#if HAVE_SCHED_SETAFFINITY
+static void ConvertPalToNativeCpuSet(const CpuSetBits& pal, cpu_set_t& native)
+{
+    memcpy(native.__bits, pal.Bits, ARRAY_SIZE(native.__bits));
+}
+
+extern "C" int32_t SchedSetAffinity(int32_t pid, CpuSetBits* mask)
+{
+    assert(mask != nullptr);
+    cpu_set_t set = {};
+    ConvertPalToNativeCpuSet(*mask, set);
+    return sched_setaffinity(pid, sizeof(cpu_set_t), &set);
+}
+#endif
+
+#if HAVE_SCHED_GETAFFINITY
+static void ConvertNativeToPalCpuSet(const cpu_set_t& native, CpuSetBits& pal)
+{
+    memset(pal.Bits, 0, ARRAY_SIZE(pal.Bits));
```

Comment:
Elsewhere I think we've written this as:
```C
pal = { };
```
Does that not work here for some reason?